lib/ts/recipe/userroles/userRoleClaim.ts

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import { PrimitiveArrayClaim } from "../session";
+
+export const UserRoleClaim = new PrimitiveArrayClaim({
+    id: "st-role",
+    refresh: async () => {
+        // Right now we can't refresh this claim from the client side automatically
+    },
```

Comment:
should you not also provide the `shouldRefresh` function to return false? Cause if that returns true, and then we call this, it will do nothing.

Reply:
The `shouldRefresh` func isn't configurable like this, and since calling this is a no-op, it's not an issue. I don't think reimplementing/overriding validators here would be worth it.